src/SprykerEco/Zed/Computop/Persistence/ComputopQueryContainerInterface.php

```diff
@@ -10,6 +10,9 @@
 interface ComputopQueryContainerInterface
 {
     /**
+     * Specification:
+     * - Returns SpyPaymentComputopQuery filtered by ID.
```

Comment:
```suggestion
     * - Returns `SpyPaymentComputopQuery` filtered by ID.
```
Please, add backticks to other specifications too.

Reply:
Done.